tests/basic_presence_test.py

```diff
@@ -17,10 +16,14 @@
     "Eighty percent of success is showing up.",
     "Life is what happens to you while you’re busy making other plans.",
     "Strive not to be a success, but rather to be of value.",
-    "The best time to plant a tree was 20 years ago. The second best time is now.",
+    (
+        "The best time to plant a tree was 20 years ago."
+        " The second best time is now."
+    ),
```

Comment:
What's going on here? Was the line too ling?

Reply:
Yep, according to `pep8` & `E501` code should be restricted as `80` characters (72 for docstring & comment).
This is getting less popular nowadays and people more often tend to go for `90`-`100` or even `120` as max length.
(i was used to `120` for a long but switched to `80`, allowing me to always have 2 python files open in split mode)

Reply:
It's less the choice of line length, more so the use of `(\n""\n""),` - I've never seen this way of splitting up a string. My go-to is `""\n\\n"",`

Reply:
> 
> 
> It's less the choice of line length, more so the use of `(\n""\n""),` - I've never seen this way of splitting up a string. My go-to is `""\n\\n"",`

The problem with using `\`, in my opinion, is that it offset the whole text depending on the variable name, wish can be a pain to handle when having a long one, see below:
```py
long_text_variable_name = "Lorem ipsum dolor sit amet, consectetur adipiscing" \
                          " elit, sed do eiusmod tempor incididunt ut labore" \
                          " et dolore magna aliqua. Ut enim ad minim veniam," \
                          " quis nostrud exercitation ullamco laboris nisi ut" \
                          " aliquip ex ea commodo consequat. Duis aute irure" \
                          " dolor in reprehenderit in voluptate velit esse" \
                          " cillum dolore eu fugiat nulla pariatur. Excepteur" \
                          " sint occaecat cupidatat non proident, sunt in" \
                          " culpa qui officia deserunt mollit anim id est" \
                          " laborum."
```

```py
long_text_variable_name = (
    "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod "
    "tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim "
    "veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea "
    "commodo consequat. Duis aute irure dolor in reprehenderit in voluptate "
    "velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat "
    "cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id "
    "est laborum."
)
```

Aslo python totally understand the string contatenation 
```py
>>> "a" "b"
'ab'
```
and directly optimize it within the compilation phase (a very tiny phase before the interpretation that checks syntax, and optimize some part such as string, ...)